lc-hax/Scripts/Patches/UnlockableSuitPatch.cs

```diff
@@ -0,0 +1,41 @@
+using HarmonyLib;
+using Hax;
+
+[HarmonyPatch(typeof(UnlockableSuit))]
+public static class UnlockableSuitPatch {
+
+    private static bool overrideSuit = false;
+    private static int customSuitID = -1;
+    private static int originalSuitID = -1;
```

Comment:
Use `__state` instead to pass state.

https://harmony.pardeike.net/articles/patching-injections.html#__state

Reply:
i never did __state before.